tests/test-cases/Group23-VIC-Machine-Service/23-04-VCH-Inspect.robot

```diff
@@ -151,3 +165,97 @@ Get Invalid VCH
 
 Get Invalid VCH Within Datacenter
     Get Path Under Target  /datacenter/${DC-ID}/vch/INVALID
+    
+Get Deleted VCH
+    Create VCH    '{"name":"%{VCH-NAME}-api-test-simple","compute":{"resource":{"name":"%{TEST_RESOURCE}"}},"storage":{"image_stores":["ds://%{TEST_DATASTORE}"]},"network":{"bridge":{"ip_range":"172.16.0.0/12","port_group":{"name":"%{BRIDGE_NETWORK}"}},"public":{"port_group":{"name":"${PUBLIC_NETWORK}"}}},"auth":{"server":{"generate":{"cname":"vch.example.com","organization":["VMware, Inc."],"size":{"value":2048,"units":"bits"}}},"client":{"no_tls_verify": true}}}'
```

Comment:
would it be cleaner if we just use the VCH deployed in the test setup? And later for complex VCH test, we just create another VCH
Also this case is using API to create a VCH, it feels like it would fall into the interoperability test category

Reply:
the test setup is using CLI to create VCH so I want to test with Create VCH using API